coordinator/src/service.rs

```diff
@@ -824,6 +824,136 @@ impl Coordinator for CoordService {
     fn metrics(&self) -> &Arc<CoordServiceMetrics> {
         &self.metrics
     }
+
+    async fn update_tags_value(
+        &self,
+        table_schema: TskvTableSchemaRef,
+        new_tags: Vec<UpdateSetValue>,
+        record_batch: RecordBatch,
+    ) -> CoordinatorResult<()> {
+        let tenant = &table_schema.tenant;
+        let db = &table_schema.db;
+        let table_name = &table_schema.name;
+        let num_rows = record_batch.num_rows();
+        let schema = record_batch.schema().fields().clone();
+        let columns = record_batch.columns();
+        let mut repl_idx: HashMap<ReplicationSet, Vec<usize>> = HashMap::new();
+
+        let meta_client = self.meta.tenant_meta(tenant.as_str()).await.ok_or(
+            CoordinatorError::TenantNotFound {
+                name: tenant.to_string(),
+            },
+        )?;
+        let db_schema =
+            meta_client
+                .get_db_schema(db.as_str())?
+                .ok_or_else(|| MetaError::DatabaseNotFound {
+                    database: db.to_string(),
+                })?;
+        let db_precision = db_schema.config.precision_or_default();
+
+        // 找到每一行应该所属的ReplicationSet
```

Comment:
can use `prune_shards` to get the vnodes of the table  to avoid finding the vnode to which each row belongs.